sound/soc/sof/intel/hda-mlink.c

```diff
@@ -1046,7 +1046,7 @@ void hdac_bus_eml_set_mic_privacy_mask(struct hdac_bus *bus, bool alt, int elid,
 
 	h2link->mic_privacy_mask = mask;
 }
-EXPORT_SYMBOL_NS(hdac_bus_eml_set_mic_privacy_mask, "SND_SOC_SOF_HDA_MLINK");
+EXPORT_SYMBOL_NS(hdac_bus_eml_set_mic_privacy_mask, SND_SOC_SOF_HDA_MLINK);
```

Comment:
No, this is not correct, see:
cdd30ebb1b9f ("module: Convert symbol namespace to string literal")

Reply:
@naveen-manohar This is only for your 6.12 kernel, not for topic/sof-dev branch

Reply:
@ujfalusi Thanks for the info.
As the build fails on v6.12, for now i have posted a DONOT_MERGE series